### ISSUE TITLE
Implement dark mode

### ISSUE THREAD

**Issue by ruthmoog** [ruthmoog/portfolio#5]
Add dark mode option for improved accessibility and learning.  Ideally the mode used should be picked up from the user's active preference and be about to toggle on/off via the UI.

Resource: [Dark mode in 5 minutes, with inverted lightness variables
](https://lea.verou.me/2021/03/inverted-lightness-variables/)
